app/views/taxFreeAmountNew.scala.html

```diff
@@ -70,6 +69,35 @@ <h2 class="flush--bottom">@Messages("tai.taxFreeAmount.detailsection.heading")</
 
                 </section>
 
+                <h2 id="addSomethingMissingHeading" class="heading">@Messages("tai.incomeTaxSummary.addMissingIncome.section.heading")</h2>
```

Comment:
Wondering if we could place the new links within the taxFreeAmountDetail <section>, as they seem to relate to the detail items semantically.
That would mean everything would be contained within one of the two existing HTML sections - summary or detail. The new header could be switched to an h3 with a class of 'heading-medium' - so it would look the same as you have it visually, but sit semantically along with the others - allowance, additions, deductions, total, something missing, and the screen reader would treat it the same, if the user navigates into the detail section.
Not a right or wrong thing - just wondering about semantics/groupings. If you reckon it's cleaner outside of the detail, no prob.